onmt/inputters/inputter.py

```diff
@@ -378,7 +391,7 @@ def load_vocabulary(vocabulary_path, tag=""):
     """
     vocabulary = None
     if vocabulary_path:
-        vocabulary = set([])
+        vocabulary = []
```

Comment:
What is the reason to use a `list` instead of a `set`?

Reply:
Same reason as above - to make sure the vocabulary is in the same sorting order as the specified file.